### ISSUE TITLE
Blue double checkmark appears although the trustlines aggreement in not signed

### ISSUE THREAD

**Issue by seichris** [trustlines-network/feedback#164]
### Basic information:

* OS (Android / iOS / both):  Android 7.0 xiaomi 5s
* Version: 1.0.18 (1532522735)

### Issue description:
The blue double checkmark appears under the Trustline Request, although the trustlines aggreement in not signed by the other party.

Is it implemented that way or a bug?

### Possible solution
I think the double blue mark should just appear if the other one signed the agreement.

**Comment by jensneuber** [comment]
Nope. The blue double checkmark indicates, that this transaction is mined and verified. If the request is signed, it will be replaced by the trustline update.

**Comment by berndbohmeier** [comment]
We should think about the way to indicate this. The blue double check marks are probably confusing as people no that from whatsapp as people have seen it.

**Comment by berndbohmeier** [comment]
The signing goes through the same steps, so when would you show the blue check marks?
The signed agreement will also be processed by the relay server, then pending on the chain, then confirmed on the chain.

**Comment by seichris** [comment]
Just show the two blue checkmarks. when the whole process is through. When the signing is confirmed on the blockchain.

**Comment by jensneuber** [comment]
We have two different events here: Trustline Update Request vs. Trustline Update, where a Trustline Update is a signed Trustline Update Request. 

@seichris I see your point. If we want to change this, we need a new concept on those events though. Your suggestion would reduce it to one event type, being differentiated by different statuses.

**Comment by berndbohmeier** [comment]
Please open a new issue in the project repository and then work out a new flow there.

**Comment by seichris** [comment]
Opened a new issue here: https://github.com/trustlines-network/project/issues/203
